src/lib/db/feature-toggle-client-store.ts

```diff
@@ -51,28 +50,16 @@ export default class FeatureToggleClientStore
 
     private logger: Logger;
 
-    private inlineSegmentConstraints: boolean;
-
     private timer: Function;
 
-    private flagResolver: IFlagResolver;
-
-    constructor(
-        db: Db,
-        eventBus: EventEmitter,
-        getLogger: LogProvider,
-        inlineSegmentConstraints: boolean,
-        flagResolver: IFlagResolver,
```

Comment:
Unrelated cleanup of unused parameters to simplify test setup src/test/e2e/stores/feature-toggle-client-store.e2e.test.ts